lnclient/ldk/ldk.go

```diff
@@ -436,7 +440,7 @@ func (ls *LDKService) Shutdown() error {
 	}
 
 	logger.Logger.Debug("Destroying LDK node object")
-	node.Destroy()
+	ls.node.Destroy()
 
```

Comment:
Should we here set `shuttingDown` to false again? So we prevent shutdown being blocked on restart/2nd attempt?

Reply:
It's an instance variable, so a new LDKNode will have `shuttingDown` as false

Reply:
Ah true :+1: